packages/webservice/app/src/test/kotlin/com/yahoo/navi/ws/test/framework/IntegrationTest.kt

```diff
@@ -75,6 +87,29 @@ abstract class IntegrationTest: RestAssuredSupport {
         }
     }
 
+    /**
+     * Execute select query and return number of rows
+     * @param query
+     * @return number of rows for a query
+     */
+    fun getCountForSelectQuery(query: String): Int {
+        var numberOfRows = 0
+        try {
+            DriverManager.getConnection(DATABASE_CONNECTION_URL, DATABASE_USER, DATABASE_PASSWORD).use{ conn ->
+                conn.createStatement().use{ stmt ->
+                    val rs = stmt.executeQuery(query)
+                    rs.last()
+                    numberOfRows = rs.getInt(1)
```

Comment:
Since this is meant for a count(*) query that won't work.  it'll always return 1 in that case which is incorrect.

I adjusted the comments to reflect this correctly as the comments did not reflect what the method did.